xarray/tests/test_backends.py

```diff
@@ -4327,7 +4332,7 @@ def test_open_mfdataset(self) -> None:
                 original.isel(x=slice(5)).to_netcdf(tmp1)
                 original.isel(x=slice(5, 10)).to_netcdf(tmp2)
                 with open_mfdataset(
-                    [tmp1, tmp2], concat_dim="x", combine="nested"
+                    [tmp1, tmp2], concat_dim="x", combine="nested", chunks={}
```

Comment:
Shouldn't this and the next few changes be unnecessary at the moment?

Reply:
Yes, these changes are required if None was the default. I suppose we can undo these and leave it for the future.